game/panorama/localization/addon_english.txt

```diff
@@ -17,4 +17,20 @@
   "ngp_pass"                        "Pass"
 
   "dota_gamemode_15"                "Open Angel Arena"
+
+  "game_info_header_objective"      "Objective:"
+  "brief_description"               "Accumulate points by killing players and bosses. First to hit the kill limit wins!"
+  "game_info_header_how_to_play"    "How To Play:"
+
+  "game_info_duels"                "Take to the <span class='duels'>Duel</span> Arena in a variety of team comps and battle to the death. Activate the highground and use the shop to get the upper-hand."
+
+  "game_info_bosses"               "<span class='bosses'>Bosses</span> and their loot are the key to unlocking powerful item upgrades. Their difficulty increases as you defeat them, and some don't play by the rules."
+
+  "game_info_farming_core"           "Killing the first Boss near your base for the first time rewards your team with a <span class='farmingcore'>Farming Core.</span> When combined with your upgraded boots, this item grants your hero special creep-killing properties."
+
+  "game_info_reflex_core"          "Killing the second Boss near your base for the first time rewards your team with a <span class='reflexcore'>Reflex Core.</span> The Reflex items are among the most powerful items in the game."
```

Comment:
`The Reflex items are among the most powerful items in the game.` -> `The Reflex items are among the most powerful in the game.`